src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -14,6 +18,39 @@ class Tribe__Tickets__Repositories__Attendee__Commerce extends Tribe__Tickets__A
 	 */
 	protected $key_name = 'tribe-commerce';
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->attendee_provider = tribe( 'tickets.commerce.paypal' );
+
+		$this->create_args['post_type'] = $this->attendee_provider->attendee_object;
+
+		// Use a regular variable so we can get constants from it PHP <7.0
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: e876ed2ca1418a83a1f196e91491d98e</sub>